readme.md

```diff
@@ -1,212 +1,234 @@
+
+* **Question:** The response you receive is _not_ the document you updated. What information is being represented here? Try replacing `Model.updateOne()` with the [Model.findOneAndUpdate()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndUpdate) and see the difference in the result.
+
+* **Your Answer:**
+
+---The response includes various metadata about the PUT request itself, such as number of documents updated, etc.
+
+* **Question:** This new method will return the _old document_ instead of the new one. What option can you add to your statement to return the new one?
+
+* **Your Answer:**
+
+--- Using the `new` argument in the `Model.findOneAndUpdate()` method will result in the updated document being returned in the response.
+
+* **Question:** Another field was updated when you ran this command. Which one was it?
+
+* **Your Answer:**
+
+--- The `updated_at` key/value pair was updated as a result of the update to the document.
+
+* **Question:** Take a look at the terminal window running your server. You are likely getting the following deprecation warning:
+  ```
+  DeprecationWarning: Mongoose: `findOneAndUpdate()` and `findOneAndDelete()` without the `useFindAndModify` option set to false are deprecated. See: https://mongoosejs.com/docs/deprecations.html#-findandmodify-
+  ```
+  Take a look [at this page](https://mongoosejs.com/docs/deprecations#findandmodify) to see how to fix it. Describe the changes that took place.
+
+* **Your Answer:**
+
+---You can resolve this issue by adding `useFindAndModify: true` to your mongoDB connection statement . If i am understanding the Mongoose documentation correctly, this statement is added in order to force Mongoose to use MongoDB driver's `findOneandUpdate()` function instead of MongoDB driver's `findAndModify()` function.  Is that correct?
+
+- [ ] Update the API so that you can successfully delete a record. Use the [Model.findOneAndDelete()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndDelete) method.
+
+---
+
+- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "_id", "title", "start_year", and "season_count" on all the above requests.
+
+* **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
+
+* **Your Answer:** 
+
+---`Model.create()` failed to return a response, even though the creation of the document was successful in the DB. I am not entirely sure that i understand the error in the terminal, but it seems as though i am dealing with a promise rejection and would need to use something like `.catch()` to handle with an error message, function, etc. Is it because ``Model.create()` is not returning a query?

```

Comment:
Correct, `Model.create()` returns a different type of object so you can't call `.select()` on it.